README.md

```diff
@@ -165,7 +165,6 @@ Instead of overwriting an entire config file just to change one value, try to
 [jenkins]: https://jenkins.ocf.berkeley.edu/view/puppet-deploy/
 [r10k]: https://github.com/puppetlabs/r10k
 [puppet-forge]: https://forge.puppet.com/
-[augeas]: http://projects.puppetlabs.com/projects/1/wiki/puppet_augeas
+[augeas]: https://puppet.com/docs/puppet/4.8/types/augeas.html
```

Comment:
Ideally this wouldn't be pinned on one version of the puppet docs, but pinning on latest wouldn't be a great solution either...

Reply:
Yeah, I think leaving it like this is fine, latest seems a bit more likely to break anyway, so I think this is slightly better.